### PR TITLE
CUST-1705  responsive config app

### PR DESCRIPTION
**Pull request by DarrylD**

### CODE REVIEW

File: src/directives/page-header-options/page-header-options.spec.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*jshint esnext: true */
+
+import PageHeader from './page-header';
```
Comment (jsmylnycky) [inline src/directives/page-header-options/page-header-options.spec.js]: I'm guessing that should be ./page-header-options?

Please fix up this file, it looks like it wasn't updated to reflect Page Header Options all the way through.

Make sure your unit tests are compiling without any jshints (they would be above the individual checkmark outputs), and that you're getting 100% coverage before submitting the fixes please.

Comment (DarrylD) [inline src/directives/page-header-options/page-header-options.spec.js]: @jsmylnycky JShint definitely didn't complain at me about this, although would had been nice it it did.

Comment (jsmylnycky) [inline src/directives/page-header-options/page-header-options.spec.js]: @DarrylD Hm ok.  I just checked it out, did an npm install to make sure everything was up to date, and ran 'gulp test:tdd' and immediately got this:

```
ERROR in ./src/directives/page-header-options/page-header-options.spec.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./page-header in /Users/jason/git/peach/v2/ngPeach/src/directives/page-header-options
 @ ./src/directives/page-header-options/page-header-options.spec.js 7:18-42
```

Then it spits out that 0 tests completed.  Can't say why you weren't seeing that message.

Comment (DarrylD) [inline src/directives/page-header-options/page-header-options.spec.js]: @jsmylnycky I was referring to the jshint in my editor. Maybe jumping between copying the dist to config and running the tests it got overlooked in the terminal. Anyway, gonna fix that up.

Comment (jsmylnycky) [inline src/directives/page-header-options/page-header-options.spec.js]: @DarrylD  Oh gotcha, yeah I cant vouch for the editor based jshint :)

**Comment by jsmylnycky** [comment]
:+1: 

Side note, merging this and https://github.com/peachworks/ngPeach/pull/37 will probably have a lot of conflicts.  Will need to come up with a proper plan to review some of the refactor in 37 and apply it here.  Don't worry about it now tho.

**Comment by darushka** [comment]
:+1:

**Comment by rynomars** [comment]
:thumbsup:

**Comment by jsmylnycky** [comment]
@DarrylD  Please merge squash

**Comment by DarrylD** [comment]
@jsmylnycky s/m d60f77332a13453b0f0b805c92d67301878b2517

**Comment by jsmylnycky** [comment]
@DarrylD  I had to revert.  After pulling the changes to prep it for build, I ran the tests and a few are failing and theres not 100% coverage.  Also, it looks like the new page-header-options component was never updated with the module syntax like the others.

Once that gets cleaned up and tests pass, we can go ahead and get this merged.

**Comment by DarrylD** [comment]
@jsmylnycky re-s/m 1a2b94366bf23f3e84b9543caecfa8c1832258fb

**Comment by jsmylnycky** [comment]
Released v0.4.3

**Pull request closed without merging**